pkg/apiserver/apiserver_test.go

```diff
@@ -57,10 +57,19 @@ func convert(obj runtime.Object) (runtime.Object, error) {
 }
 
 // This creates fake API versions, similar to api/latest.go.
-const testVersion = "version"
-const newVersion = "version2"
-
-var versions = []string{testVersion, newVersion}
+var testAPIGroup = "test.group"
+var testGroupVersion = unversioned.GroupVersion{Group: testAPIGroup, Version: "version"}
+var testVersion = testGroupVersion.String()
```

Comment:
> Can we remove the testVersion? It's containing group/version.

This intersects with the `RESTMapping` that I'm touching now.  May I do it as part of that pull?  The use of `version` in and out of the `RESTMapper` was inconsistent between tests and code